instruments/lakeshore/lakeshore475.py

```diff
@@ -100,14 +102,14 @@ def field_units(self):
 
         Acceptable units are Gauss, Tesla, Oersted, and Amp/meter.
 
-        :type: `~quantities.unitquantity.UnitQuantity`
+        :type: `~pint.Unit`
         """
         value = int(self.query('UNIT?'))
         return LAKESHORE_FIELD_UNITS[value]
 
     @field_units.setter
     def field_units(self, newval):
-        if isinstance(newval, u.unitquantity.UnitQuantity):
+        if isinstance(newval, pint.Unit):
```

Comment:
It would not, since `type(u.gauss) != type(1 * u.gauss)` where the first is of type `pint.Unit` and the second is of type `pint.Quantity`.

Granted, would could switch it to `isinstance(newval, u.Unit)`